### ISSUE TITLE
IOS > 15.2 and restricted users -> blank config

### ISSUE THREAD

**Issue by zxyz** [ytti/oxidized#677]
thank you for Oxidized! 

Disclaimer: I'm new to IOS and probably not aware of the further implications this might have.

As explained in [1] there is a problem with IOS 15.2 and a blank config for non level-15 users. 
With the following I also only got a blank config:

> privilege exec level 10 show running-config view full
> privilege exec level 10 show running-config view
> privilege exec level 10 show running-config

Adding  `file privilege 10`  (see [1]), I got `show running-config view full` working for me, but with  `show running-config` I only got the short config. So I changed the  according line in the Oxidized **ios.rb** file

-`show running_config`
+`show running-config view full`

I have no idea of the implications of changing this upstream. Nevertheless this might help someone in the future.

[1] http://commandline.ninja/2016/05/30/rancid-and-ios-15-2-blank-config-and-how-to-work-around-newer-file-privileges/

**Comment by ytti** [comment]
We may need to guard  this in vars, perhaps something like

```
post do
  cmd_line = "show run"
  cmd_line += " view full" if vars(:ios_rbac)

  cmd cmd_line do |cfg|
     ...
  end
end
```

**Comment by faekz0r** [comment]
> We may need to guard this in vars, perhaps something like
> 
> ```
> post do
>   cmd_line = "show run"
>   cmd_line += " view full" if vars(:ios_rbac)
> 
>   cmd cmd_line do |cfg|
>      ...
>   end
> end
> ```

Where exactly would I need to define ios_rbac in the model file and what should be the value for it in the router.db to trigger the " view full" addition?

I tried:
- changing vars: {} to vars: { ios_rbac }
- changing vars: {} to vars: { ios_rbac: true }
- vars_map with "ios_rbac: x" and "true" in router.db

Which of none worked of course.

**Comment by derekivey** [comment]
I ran into this issue as well. Can this feature be merged in?

**Comment by derekivey** [comment]
This was merged in #2588. This issue can be closed.
